taskt/Core/Automation/Commands/Outlook Commands/MoveCopyOutlookEmailCommand.cs

```diff
@@ -17,23 +17,15 @@ namespace taskt.Core.Automation.Commands
     [Group("Outlook Commands")]
     [Description("This command moves or copies selected emails in Outlook.")]
 
-    public class MoveCopyOutlookEmailsCommand : ScriptCommand
+    public class MoveCopyOutlookEmailCommand : ScriptCommand
     {
         [XmlAttribute]
-        [PropertyDescription("Source Mail Folder Name")]
-        [InputSpecification("Enter the name of the Outlook mail folder the emails are located in.")]
-        [SampleUsage("Inbox || {vFolderName}")]
+        [PropertyDescription("MailItem")]
+        [InputSpecification("Enter the MailItem to reply to.")]
```

Comment:
Please update the `InputSpecification` according to the Command Description.

Reply:
done